druid/src/test/java/org/apache/calcite/test/DruidAdapterIT.java

```diff
@@ -577,23 +577,28 @@ protected boolean enabled() {
    */
   @Test public void testCastToTimestamp() {
     final String sql = "select cast(\"timestamp\" as timestamp) from \"foodmart\"";
-    final String druidQuery = "{'queryType':'scan','dataSource':'foodmart',"
-        + "'intervals':['1900-01-09T00:00:00.000Z/2992-01-10T00:00:00.000Z']',"
-        + "'virtualColumns:[{'type':'expression','name':'vc',"
-        + "'expression':'timestamp_parse("
-        + "timestamp_format(\"__time\",'yyyy-MM-dd\\u0027T\\u0027HH:mm:ss.SSS\\u0027Z\\u0027',"
-        + "'America/New_York'),'yyyy-MM-dd\\u0027T\\u0027HH:mm:ss.SSS\\u0027Z\\u0027','UTC')',"
-        + "'outputType':'LONG'}],"
-        + "'columns':['vc'],"
-        + "'resultFormat':'compactedList'}";
+    final String druidQuery = "{\"queryType\":\"scan\",\"dataSource\":\"foodmart\","
+        + "\"intervals\":[\"1900-01-09T00:00:00.000Z/2992-01-10T00:00:00.000Z\"],"
+        + "\"virtualColumns\":[{\"type\":\"expression\",\"name\":\"vc\","
+        + "\"expression\":\"timestamp_parse("
+        + "timestamp_format(\\\"__time\\\",'yyyy-MM-dd\\\\u0027T\\\\u0027HH:mm:ss.SSS\\\\u0027Z\\\\u0027',"
+        + "'America/New_York'),'yyyy-MM-dd\\\\u0027T\\\\u0027HH:mm:ss.SSS\\\\u0027Z\\\\u0027','UTC')\","
+        + "\"outputType\":\"LONG\"}],"
+        + "\"columns\":[\"vc\"],"
+        + "\"resultFormat\":\"compactedList\"}";
 
     CalciteAssert.that()
          .enable(enabled())
```

Comment:
Here we have 5 spaces :(, but we use 4 spaces for continuous indention.

Reply:
I updated the test with the shortened query string but I'm not sure what the spacing problem is. Checkstyle compiles my code just fine. The continuous integration fails because SqlOperatorBaseTest.testSimilarToOperator fails the test for JDK 13

Reply:
JDK 13 is failing because of some other problems[1], but we still have to fix indentions if we found some. There are 5 whitespaces in front of `.enable(enabled())` :)

[1] https://lists.apache.org/thread.html/f5fe7944794a546e3bc27d2c059bc7134bba10b3973ced42df9f707a@%3Cdev.calcite.apache.org%3E

Reply:
Ok I fixed the indentation level